pages/selfCheckQuestions.tsx

```diff
@@ -0,0 +1,322 @@
+        setCount(newCount);
+        setList(newList);
+    }
+
+    function removeOneQuestion(id) {
+        const newList = list.filter((item) => item._id !== id);
+        setList(newList);
+    }
+    function removeAllQuestions() {
+        const newList = [];
+        setList(newList);
+    }
+
+    const selfCheckQuestionSize = list.length;
+
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    return (
+        <Flex direction="column" minH="100vh">
+            <Header />
+
+            <Flex mt={10} wrap={"wrap"} justifyContent={"left"} width={"full"}>
+                <Text mx={10} fontWeight="bold" fontSize="4xl">
+                    Create a Tool
+                </Text>
+                <Spacer />
+                <ButtonGroup wrap={"wrap"} spacing={"1.5rem"} mx={12}>
+                    <Button
+                        _hover={{ bg: "#F3F3F3" }}
+                        _active={{
+                            transform: "scale(0.95)",
+                        }}
+                        onClick={onOpen}
+                        minWidth={"90"}
+                        colorScheme="white"
+                        variant="outline"
+                    >
+                        Discard
+                    </Button>
+                    <Button
+                        _hover={{ bg: "#121310" }}
+                        _active={{
+                            transform: "scale(0.95)",
+                        }}
+                        minWidth={"90"}
+                        color="white"
+                        background="black"
+                        variant="outline"
+                        onClick={() => {
+                            alert("Check the console for the object");
+                            console.log(list);
+                        }}
+                    >
+                        Save
+                    </Button>
+                </ButtonGroup>
+            </Flex>
+            <Modal
+                blockScrollOnMount={false}
+                isOpen={isOpen}
+                onClose={onClose}
+                motionPreset="slideInBottom"
+            >
+                <ModalOverlay />
+                <ModalContent p={"5"}>
+                    <ModalHeader>Delete Tool </ModalHeader>
+                    <ModalCloseButton />
+                    <ModalBody>
+                        Are you sure you want to discard this tool? This is a
+                        permanent action that cannot be undone.
+                    </ModalBody>
+                    <ModalFooter>
+                        <Button
+                            variant="outline"
+                            colorScheme="black"
+                            mr={3}
+                            w={100}
+                            onClick={onClose}
+                        >
+                            Cancel
+                        </Button>
+                        <Button
+                            onClick={() => {
```

Comment:
Might have to updated when integrating this change into the actual Tool builder page